cmd/renterd/main.go

```diff
@@ -201,11 +217,27 @@ func main() {
 
 	workerAddr, workerPassword := workerCfg.remoteAddr, workerCfg.apiPassword
 	if workerAddr == "" {
-		w, cleanup, err := node.NewWorker(workerCfg.WorkerConfig, bc, getWalletKey())
+		workerDir := filepath.Join(*dir, "worker")
+		if err := os.MkdirAll(workerDir, 0700); err != nil {
+			log.Fatal(err)
+		}
+
+		workerLog := filepath.Join(workerDir, workerCfg.WorkerConfig.ID)
+		l, closeFn, err := node.NewLogger(workerLog)
```

Comment:
I considered that but that does not really work well in a scenario where the `worker` and `bus` run on different machines. I think that because of that we have to split it up into logs per sub-system, like we can't have a `renterd.log` in a single server setup, and separate logs in multi node setups.

Reply:
ok, will update